near/res/test_records.json

```diff
@@ -0,0 +1,98 @@
+[
```

Comment:
what is this file?

Reply:
Test `testnet_genesis` assert a >0 total supply, which is calculated from sum in records, so this small fixture records is needed

Reply:
well let's remove that check then. Having this random file is just weird.

Reply:
OK